pegasus/sites.v3/code.org/views/csp_standalone_units.haml

```diff
@@ -0,0 +1,39 @@
+.tile-container-responsive{style: "margin-top: 20px;"}
+  .col-50
+    .tutorial-tile
+      %a{href: CDO.studio_url("/s/devices")}
+        %img.tutorial-tile-img{src: "/images/fill-473x258/maker/maker-graphic.png", alt: ""}
+      .tutorial-info
+        %span
+          %h3.tutorial-info-h
+            Creating Apps with Devices
+          .smalltext
+            Every day, we interact with devices that shape our world. In this module, students will learn how to program devices to take in inputs and create new outputs.
+        %span
+          %a{href: CDO.studio_url("/s/devices")}
+            %button.tutorial-info-start.tutorial-gray
+              Explore Module
+          .tutorial-info-guide
+            %a{href: "/maker"}
+              More About Maker
```

Comment:
Nit: is this block supposed to be the same height as the AI block? Usually our "course cards" are the same height, e.g. https://studio.code.org/courses but not sure if it's important

Reply:
I updated this to make them the same height — I did inline styles since these will vary based on the content in them, but I made them both the same height.

Reply:
Excellent!